main.go

```diff
@@ -350,6 +361,103 @@ func main() {
 	if err := uAPI.Endpoint.RegisterMethod(fmt.Sprintf("%s/{id}.png", imageHandlerPath), http.MethodGet, "public", handler.imagesHandler); err != nil {
 		log.Fatal(err)
 	}
+	// if a bot FID is provided, start the bot background process
+	if botFid > 0 {
```

Comment:
Please move this to the bot package. The only thing that would go here is the registration of the HTTP handler (which would point to bot.WebHookHandler or something like this).